src/content/docs/style-guide/word-choice/pricing-language-guidelines.mdx

```diff
@@ -12,15 +12,26 @@ This doc explains required styles and recommended phrasings for pricing-related
 
 If you haven't already, please read [Overview of pricing model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model) to understand our two different pricing models. Understanding that we have two different pricing models is an important first step of understanding our guidelines around pricing-related language. 
 
-These are the primary places in our public docs that pricing- and billing-related impacts are documented:
+These are the primary places in our public docs where pricing- and billing-related impacts are documented:
 
 * We explain billing impacts in the main pricing and billing-related docs. The main docs in this area are: 
     * [New Relic One pricing model](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/)
     * [Original pricing model](/docs/accounts/original-accounts-billing/product-pricing/product-based-pricing)
 * When a feature requires a pricing edition (Standard, Pro, Enterprise), we include that in the requirements of the doc for that feature. But because there are only a few capabilities limited by the Pro and Enterprise editions, this is an infrequent need.
-* User type: information about how features are restricted based on user type is located in the [user type doc](/docs/accounts/accounts-billing/new-relic-one-user-management/user-type). For more about style and formatting related to user type, see [User-related style guidelines](/docs/style-guide/word-choice/user-related-language-guidelines).
+* User type (basic, core, and full platform): information about how features are restricted based on user type is located in the [user type doc](/docs/accounts/accounts-billing/new-relic-one-user-management/user-type). For more about style and formatting related to user types, see [User-related style guidelines](/docs/style-guide/word-choice/user-related-language-guidelines).
 * Our [license docs](https://docs.newrelic.com/docs/licenses) (such as [this one](/docs/licenses/license-information/usage-plans/new-relic-one-usage-plan-descriptions)): these docs are primarily written by legal- and agreement-focused teams, not the docs team. 
 
+### Guidelines for features that may result in high data generation [#high-data-ingest]
+
+Some New Relic tools and features can result in high levels of data ingest and, if a customer is on the New Relic One pricing model, higher billing. Some guidelines for instances where it's thought that the possibility for spikes in data ingest should be addressed in the documentation: 
```

Comment:
I think you can delete "Some guidelines...in the documentation" in line 26, because line 28 says almost the exact same thing.